example/integration_test/util/assertion_helper.dart

```diff
@@ -3,11 +3,23 @@ import 'package:flutter/cupertino.dart';
 import 'package:flutter_test/flutter_test.dart';
 import '../extensions/string_extension.dart';
 
-// Assert expected text in a Text widget whose key starts with "nativeResponse_".
+/// Assert text in a Text widget whose key starts with "nativeResponse_" matches the expected value being passed.
 Future<void> assertNativeMessage(String suffix, String expected) async {
+  final actual = extractTextFromWidget(suffix);
+  assert(actual == expected, "Actual: $actual\nExpected: $expected");
+}
+
+/// Assert text in a Text widget whose key starts with "nativeResponse_" starts with the expected value being passed.
+Future<void> assertNativeMessageStartsWith(String suffix, String expected) async {
+  final actual = extractTextFromWidget(suffix);
+  assert(actual.startsWith(expected), "Actual: $actual\nExpected: $expected");
+}
+
+/// Return text from Text widget whose key starts with "nativeResponse_" and ends with suffix.
+String extractTextFromWidget(String suffix) {
   final testKey = Key("nativeResponse_$suffix");
   final finder = find.byKey(testKey);
+  expect(finder, findsOneWidget);
   final text = finder.evaluate().single.widget as Text;
-  final actual = text.data!.removeNewLines();
-  assert(actual == expected, "Actual: $actual\nExpected: $expected");
-}
+  return text.data!.removeNewLines();
+}
```

Comment:
please add an empty line at the end of the file

Reply:
done